config/initializers/constants.rb

```diff
@@ -6,3 +6,12 @@ module RequestHeader
   ACCEPT_JSON = {"Accept" => "application/json"}
   CONTENT_TYPE_JSON = {"Content-Type" => "application/json"}
 end
+
+module SMSNotifications
```

Comment:
Could this be SMSBodyText without the additional modules to keep it shorter? so `SMSBodyText::account_activation_msg` in usage. I'd like to extend this module with other SMS bodies once this PR is merged.